documentation/building.md

```diff
@@ -0,0 +1,7 @@
+# Using x.py
+
+The intended way to build loadstone is using the x.py script in the root of the project. For usage information do `./x.py help SUBCOMMAND`.
+
```

Comment:
What's the reason for calling it `x.py`?

Reply:
its named after a similar tool in the `rustc` repo, short for execute. wasn't sure on a proper name so i just went with it